docs/development-guide/ReadWriteSplittingPluginPerformanceResults.md

```diff
@@ -0,0 +1,8 @@
+# Read-Write Splitting Plugin Performance Results
+
+## Benchmarks
+| Benchmark                                                     | Score   | Units  |
+|---------------------------------------------------------------|---------|--------|
+| No Plugin Enabled                                             | 110.551 | sec/op |
```

Comment:
I'd like to see an overhead numbers instead of raw performance numbers provided by benchmark tests. Something that look like this (numbers are not real):

No Plugin Enabled: 0 (baseline)
Read-Write Splitting Plugin Enabled: +30 ms
Read-Write Splitting Plugin and Reader Load Balancing Enabled: +50 ms